build.assets/Makefile

```diff
@@ -220,3 +221,9 @@ release-centos6-fips: bbox-centos6-fips
 release-windows: bbox
 	docker run $(DOCKERFLAGS) -i $(NOROOT) $(BBOX) \
 		/usr/bin/make release -e ADDFLAGS="$(ADDFLAGS)" OS=windows
+
+.PHONY:pam
+pam:
+	make -C ../modules/pam_teleport
+	rm pam/* || mkdir -p pam
+	cp ../modules/pam_teleport/pam_teleport.so ../modules/pam_teleport/policy/* pam/
```

Comment:
As this is the Makefile in `build.assets`, the PAM build commands here should arguably be run inside a Docker container to reduce the need to have `pam-devel` or similar packages installed on the host.

Reply:
Great idea!
Moved `modules/pam_teleport` to `build.assets/pam` and now it'll compile `pam_teleport.so` as part of buildbox creation.